internal/compiler/builtins.slint

```diff
@@ -440,6 +440,7 @@ export component NativeSlider {
     in-out property <float> value;
     in property <float> minimum;
     in property <float> maximum: 100;
+    in property <bool> is-vertical: false;
```

Comment:
On other `bool` properties the names are defined without the`is-` suffix like `enabled`, `primary`, `pressed`. For consistent in the API I think it would make sense to name the property just `vertical`.